plugins/dell-dock/fu-dell-dock-hub.c

```diff
@@ -111,6 +111,7 @@ fu_dell_dock_hub_write_fw (FuDevice *device,
 	}
 
 	/* dock will reboot to re-read; this is to appease the daemon */
+	fu_device_set_status (device, FWUPD_STATUS_DEVICE_RESTART);
```

Comment:
Actually the device doesn't restart at the end, it restarts when you unplug it from the system.

Reply:
nevermind, I understand now